portfolio/src/main/webapp/project.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Project</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div class="nav" id="mynav">
+		<a href="index.html">Home</a>
+		<a class="active" href="project.html">Projects</a>
+		<a href="experience.html">Experiences</a>
+		<a href="javascript:vohid(0);" class="icon" onclick="myFunction()">
```

Comment:
Where is `vohid` defined?

Reply:
Sorry, this is a typo, suppose to be void(0) for not doing anything, but I'm changing the syntax to href=#.

Reply:
It's a very good habit (not necessarily that any of us always succeed) to review a change (`git diff`) before committing and test everything that's altered. 🙂

Reply:
I will keep that in mind. Thanks.

Reply:
The corrected version is pushed under the correct branch now.